src/vectorchord/usage/measure-recall.md

```diff
@@ -22,12 +30,114 @@ SELECT vchordrq_evaluate_query_recall(query => $$
 $$, exact_search => true);
 ```
 
+## Query sampling <badge type="tip" text="since v0.5.2" />
+
+You may find it more useful to evaluate user-generated queries than randomly generated ones. VectorChord includes a built-in method to sample and store recent user queries. These records are grouped by index, making them easily accessible and seamlessly integrable with recall evaluation.
+
+```sql
+-- Enable the query sampling
+ALTER SYSTEM SET vchordrq.query_sampling_enable = on;
+
+-- Set the maximum number of queries to record per index
+ALTER SYSTEM SET vchordrq.query_sampling_max_records = 1000;
+
+-- Set the sampling rate (e.g., 1.0 for all queries, 0.01 for 1%)
+-- For high-traffic systems, a rate of 0.01 or lower is recommended.
+ALTER SYSTEM SET vchordrq.query_sampling_rate = 1;
+SELECT pg_reload_conf();
+
+-- Now, execute an example vector search
+-- SELECT * FROM items ORDER BY embedding <-> '[3,1,2]' LIMIT 10;
+```
```

Comment:
1. Referring to other documents, a descriptive paragraph is needed here.
2. Configuring parameters with a global setting is not intended. Please set GUC only when necessary.